src/tests/mock/pybitmessage/queues.py

```diff
@@ -0,0 +1,55 @@
+# pylint: disable=too-many-lines,import-error,no-name-in-module,unused-argument
```

Comment:
Try to find a workaround that doesn't require copy&paste or modifying production code.

Reply:
after #1916 we can symlink